lib/xsd/schema.rb

```diff
@@ -19,11 +22,12 @@ class Schema < ApplicationService
       secDNS
     ].freeze
 
-    attr_reader :xsd_schemas, :for_prefix
+    attr_reader :xsd_schemas, :for_prefix, :for_version
 
     def initialize(params)
       schema_path = params.fetch(:schema_path, SCHEMA_PATH)
       @for_prefix = params.fetch(:for_prefix)
+      @for_version = params[:for_version] || '1.1'
```

Comment:
How about `@for_version = params.fetch(:for_version, '1.1')`? https://apidock.com/rails/v4.0.2/ActionController/Parameters/fetch

It will help to maintain consistent codestyle, I suppose.